Makefile

```diff
@@ -11,7 +11,7 @@ clean:
 
 install: libdropbox_ext4.so
 	install $^ $(INSTALL_DIR)/lib/ && \
-	echo -e "#!/bin/bash\n\nLD_PRELOAD=$(INSTALL_DIR)/lib/$^ exec /usr/bin/dropbox \"\$$@\"" > $(INSTALL_DIR)/bin/dropbox && \
+	cp dropbox $(INSTALL_DIR)/bin/dropbox && \
```

Comment:
Could you change it from `cp` to `install`? That's technically the right way - and it'll keep permissions, so you can then delete the `chmod` below.

Reply:
I haven't used `install` before, but it makes sense. I've made the change.